wasm/config.js

```diff
@@ -0,0 +1,4 @@
+window.onbeforeunload = (e) => {
+    Module.ccall('emsForceSettingsSave', 'void', [], []);
+    return "good";
```

Comment:
do we need return?

Reply:
not in this case, but mb in future we want ask for confirmation if scene have changes
removed